velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -287,6 +286,35 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, ifThen) {
   assertPlanConversion(plan, "SELECT if (c0=1, c0 + 1, c1 + 2) as x FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, subField) {
+  VectorMaker vectorMaker_{pool_.get()};
```

Comment:
VectorMaker is available as a member variable of the base class. No need to create one. Use makeRowVector and makeFlatVector methods.